packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -355,8 +365,6 @@ class WritingFlow extends Component {
 			<div className="editor-writing-flow block-editor-writing-flow">
 				<div
 					ref={ this.bindContainer }
-					onKeyDown={ this.onKeyDown }
```

Comment:
yes, when we use Portals, we have parts of the toolbars that are rendered in a completely different React Tree which means events from these components won't be caught. But for WritingFlow and NavigableContainer, the behavior is tied to the DOM tree and not the React tree. so instead of using React Events, I'm relying on Native events instead.